eel/src/task_manager.rs

```diff
@@ -181,13 +181,13 @@ impl TaskManager {
                                 host: new_lsp_info.node_info.address,
                             };
                             if let Err(e) = connect_peer(&peer, peer_manager).await {
-                                error!("Connecting to peer {} failed: {}", peer, e);
+                                error!("Connecting to peer {peer} failed: {e}");
                             }
                         }
                         Some(config.success_period)
                     }
                     Err(e) => {
-                        error!("Failed to query LSP, retrying in 10 seconds: {}", e);
+                        error!("Failed to query LSP, retrying in 10 seconds: {e}");
```

Comment:
Just to take advantage of this line being edited:
```suggestion
                        error!(
                            "Failed to query LSP, retrying in {} seconds: {e}",
                            config.failure_period.as_secs()
                        );
```